### PR TITLE
Added say command

### PR DESCRIPTION
**Pull request by bealsbe**

### CODE REVIEW

File: Floofbot/Modules/Utilities.cs [hunk 1]
```diff
@@ -67,14 +71,26 @@ public async Task Avatar([Remainder] IGuildUser user = null)
...
                 ImageUrl = avatarUrl,
-                Color = Color.Magenta
+                Color = EMBED_COLOR
 
             };
             await Context.Channel.SendMessageAsync("", false, builder.Build());
+        }
 
+        [Command("say")]
+        [Summary("Reapeats a message")]
```
Comment (iShauny) [inline Floofbot/Modules/Utilities.cs:85]: typo

File: Floofbot/Modules/Utilities.cs [hunk 2]
```diff
@@ -67,14 +71,26 @@ public async Task Avatar([Remainder] IGuildUser user = null)
...
+        public async Task RepeatMessage([Remainder] string message =null)
+        {
+            EmbedBuilder builder = new EmbedBuilder()
+            {
+                Description = message,
```
Comment (iShauny) [inline Floofbot/Modules/Utilities.cs:90]: I would handle null messages. Also wary of @ tags in embeds.

Comment (bealsbe) [reply Floofbot/Modules/Utilities.cs:90]: The message is embed.  @everyone and other pings do not go through

File: Floofbot/Modules/Utilities.cs [hunk 3]
```diff
@@ -85,12 +85,19 @@ public async Task Avatar([Remainder] IGuildUser user = null)
...
+                EmbedBuilder builder = new EmbedBuilder()
+                {
+                    Description = message,
+                    Color = EMBED_COLOR
+                };
+                await Context.Channel.SendMessageAsync("", false, builder.Build());
+            }
+            else
+            {
+                await Context.Channel.SendMessageAsync("Usage: `.say [message]);
```
Comment (iShauny) [inline Floofbot/Modules/Utilities.cs:99]: missing: `"

unexpected EOF

Comment (bealsbe) [reply Floofbot/Modules/Utilities.cs:99]: whoops

**Pull request merged**